site/content/docs/5.2/components/collapse.md

```diff
@@ -146,7 +146,7 @@ const collapseList = [...collapseElementList].map(collapseEl => new bootstrap.Co
 {{< bs-table "table" >}}
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
-`parent` | selector, jQuery object, DOM element | `false` | If parent is provided, then all collapsible elements under the specified parent will be closed when this collapsible item is shown. (similar to traditional accordion behavior - this is dependent on the `card` class). The attribute has to be set on the target collapsible area. |
```

Comment:
Not sure if I should remove `jQuery object` but it's its only reference in doc.

Reply:
if it is as you say, you may remove it and in another PR we can use a term like `resolvable element` with the meaning of `selector, jQuery object, DOM element`